students/AndrewMiotke/lesson01/assignment/tests/test_unit.py

```diff
@@ -0,0 +1,100 @@
+""" Unit tests for inventory management """
+
+import unittest
+from unittest.mock import patch
+from inventory_management.inventory_class import Inventory
+from inventory_management.furniture_class import Furniture
+from inventory_management.electric_appliance_class import ElectricAppliances
+# from inventory_management.market_prices import market_price
+from inventory_management.main import main_menu
+
+class InventoryTest(unittest.TestCase):
+    """ Test the Inventory class """
+
+    def test_init(self):
+        """ Test init """
+
+        test_inventory = Inventory(123, "chair", 100, 50)
+
+        self.assertEqual(test_inventory.product_code, 123)
+        self.assertEqual(test_inventory.description, "chair")
+        self.assertEqual(test_inventory.market_price, 100)
+        self.assertEqual(test_inventory.rental_price, 50)
+
+
+    def test_return_as_dictionary(self):
+        """ Test the return as dictionary method """
+
+        test_inventory = Inventory(123, "chair", 100, 50).return_as_dictionary()
+
+        self.assertEqual(test_inventory['product_code'], 123)
+        self.assertEqual(test_inventory['description'], "chair")
+        self.assertEqual(test_inventory['market_price'], 100)
+        self.assertEqual(test_inventory['rental_price'], 50)
+
+
+class FurnitureTest(Inventory, unittest.TestCase):
```

Comment:
Looks like you have this Inventory here as a parent, you will want to remove it or you will see errors.